src/seedu/addressbook/AddressBook.java

```diff
@@ -365,7 +365,8 @@ public static String executeCommand(String userInputString) {
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
         final String[] split =  rawUserInput.trim().split("\\s+", 2);
```

Comment:
`split` is not a particularly good variable name; you could have changed it to be more reflective of what it is